sdk/storage/azblob/container/models.go

```diff
@@ -329,3 +332,79 @@ func formatTime(c *SignedIdentifier) error {
 
 	return nil
 }
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+// BatchBuilder is used for creating the batch operations list. It contains the list of either delete or set tier sub-requests.
+// NOTE: All sub-requests in the batch must be of the same type, either delete or set tier.
+type BatchBuilder struct {
```

Comment:
The convention is to keep the type definition and its methods together in the same file.

Since there's a decent amount of code for `BatchBuilder` I would suggest placing it all into a `batch_builder.go` file.

Reply:
Done